src/Behavioral.Automation/FluentAssertions/Abstractions/IAssertionBuilderWithBehaviour.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Behavioral.Automation.Elements;
+using System;
+
+namespace Behavioral.Automation.FluentAssertions.Abstractions
+{
+    public interface IAssertionBuilderWithBehaviour
+    {
+        IAssertionBuilderWithBehaviour Assert<T>(AssertionObject<T> assertion);
+        IAssertionBuilderWithBehaviour Assert<TVal>(Func<IWebElementWrapper, TVal> valueAcessor, Func<TVal, TVal, bool> comparer, TVal value, string message);
+        IAssertionBuilder And { get; }
```

Comment:
the root cause of having same 'And' properties in two different interfaces is that now we have not defined terminal operators for our calls chain. The case is that 'And' should be applied only after terminal operation and, to implement this, all terminal operators should return some dedicated interface.

Reply:
Done

Reply:
Done